backend/cmd/cleanup/moneycollector/moneycollector.go

```diff
@@ -0,0 +1,77 @@
+package moneycollector
+
+import (
+	"kubecloud/internal"
+	"kubecloud/models"
+	"sync"
+
+	"github.com/rs/zerolog/log"
+	substrate "github.com/threefoldtech/tfchain/clients/tfchain-client-go"
+)
+
+type MoneyCollector struct {
+	db              models.DB
+	config          internal.Configuration
+	substrateClient *substrate.Substrate
+}
+
+const (
+	MinBalanceThreshold = 1e5
```

Comment:
Will this amount be enough to cover the transaction fees? I think at least 1e3 or more to be safe
```suggestion
	MinBalanceThreshold = 1e2
```